### PR TITLE
[DOCS-7300] Add note for log archives

### PR DESCRIPTION
**Pull request by maycmlee**
### What does this PR do? What is the motivation?

Adds a note for log archives. PM request.

[DOCS-7300](https://datadoghq.atlassian.net/browse/DOCS-7300)

### Merge instructions

- [ ] Please merge after reviewing

### Additional notes

[DOCS-7300]: https://datadoghq.atlassian.net/browse/DOCS-7300?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/logs/log_configuration/archives.md [hunk 1]
```diff
@@ -75,6 +75,10 @@ Set up the [Google Cloud integration][1] for the project that holds your GCS sto
 
 ### Create a storage bucket
 
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">Sending logs to an archive is outside of the Datadog GovCloud environment, which is outside the control of Datadog. Datadog shall not be responsible for any logs that have left the Datadog GovCloud environment, including without limitation, any obligations or requirements that Customer may have related to FedRAMP, DoD Impact Levels, ITAR, export compliance, data residency or similar regulations applicable to such logs.</div>
```
Comment (brett0000FF) [inline content/en/logs/log_configuration/archives.md:79]: Using `Customer` sounds strange to me, but I'm guessing this is just legalese we want to use?
`...any obligations or requirements that Customer may have related to FedRAMP...`

**Review by brett0000FF — APPROVED** [review]

File: content/en/logs/log_configuration/archives.md (hunk 1 above)
Comment (maycmlee) [reply content/en/logs/log_configuration/archives.md:79]: Oh that's a good point. I'll change it to `the user`. cc @LucasC

Comment (maycmlee) [inline content/en/logs/log_configuration/archives.md:79]:
```suggestion
<div class="alert alert-warning">Sending logs to an archive is outside of the Datadog GovCloud environment, which is outside the control of Datadog. Datadog shall not be responsible for any logs that have left the Datadog GovCloud environment, including without limitation, any obligations or requirements that the user may have related to FedRAMP, DoD Impact Levels, ITAR, export compliance, data residency or similar regulations applicable to such logs.</div>
```

**Comment by LucasC** [comment]
LGTM. Thank you

**Pull request merged**